packages/pn-personafisica-webapp/src/components/Notifications/FilterNotificationsFormBody.tsx

```diff
@@ -2,7 +2,7 @@ import { FormikErrors, FormikTouched, FormikValues } from 'formik';
 import { ChangeEvent, Fragment } from 'react';
```

Comment:
restore the old version but keep the helperText for the date component

Reply:
done